e2e_tests/helpers.py

```diff
@@ -3,16 +3,33 @@
 import uuid
 from pathlib import Path
 from subprocess import run
+from time import sleep
 from typing import Generator, Optional, Tuple
 
 import pytest
+from attr import dataclass
+from cv2 import exp
 
 from darwin.exceptions import DarwinException
 from e2e_tests.objects import E2EDataset
 from e2e_tests.setup_tests import create_random_image
 
 
-def run_cli_command(command: str, working_directory: Optional[str] = None, yes: bool = False) -> Tuple[int, str, str]:
+@dataclass
+class CLIResult:
```

Comment:
Heh, funnily enough, this is roughly how I originally did it, and I also used a `namedtuple` at one point on my own version of this.